homeassistant/components/wiffi/.translations/en.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We've renamed the translations directory from `.translations` to `translations`. Please rebase on latest dev branch and rename the directory.